RELEASE/scripts/sl_ascend.ash

```diff
@@ -13888,6 +13904,12 @@ boolean sl_tavern()
 			set_property("choiceAdventure514", "1");
 		}
 
+		// Asdon usage increases Rat King chance by 8.3%
+		if(monster_level_adjustment() <= 299)
```

Comment:
I know it's better when doing a manual run to go well over 150 ml for this, but I'd rather sl_ascend not intentionally do so just because the end result is probably going to be getting beaten up a lot, especially with fewer iotm or skills. Can you change it to `< 125` instead of `<= 299`? Then again maybe with it just being rats it wouldn't be so bad... If you're relatively low iotm and/or skill and your testing doesn't get you beaten up a bunch, I'm okay with it I guess.

Reply:
I was thinking about this and the oil cartel while writing this. I was also a bit worried. I noticed sl_ascend doesn't pile on the ML much, so I wasn't massively worried. I decided to just keep it simple. My other thought was to talk with you about adding an sl_Bravery variable that if false won't try for the higher ML.

Reply:
You should probably exclude Ed from stuff like this as he doesn't deal with ML well (especially with no IotMs) and also doesn't care about getting tangles since he doesn't need to do the Pyramid.